app/assets/javascripts/controllers/container_live_dashboard/container_live_dashboard_controller.js

```diff
@@ -163,13 +162,13 @@ miqHttpInject(angular.module('containerLiveDashboard', ['ui.bootstrap', 'pattern
       });
     };
 
-    dash.refresh_graph = function(metric_id, n) {
+    dash.refresh_graph = function(metric_id, metric_type, n) {
       // TODO: replace with a datetimepicker, until then add 24 hours to the date
       var ends = dash.timeFilter.date.valueOf() + 24 * 60 * 60;
       var diff = dash.timeFilter.time_range * dash.timeFilter.range_count * 60 * 60 * 1000; // time_range is in hours
       var starts = ends - diff;
       var bucket_duration = parseInt(diff / 1000 / 200); // bucket duration is in seconds
-      var params = '&query=get_data&metric_id=' + metric_id + '&ends=' + ends +
+      var params = '&query=get_data&type=' + metric_type + 's&metric_id=' + metric_id + '&ends=' + ends +
```

Comment:
If so .. I think doing it here is a Bad Idea, but I don't have any specific complaint about it, so feel free to ignore